edsnlp/utils/numbers.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
```

Comment:
Nice 👍 we could maybe try to merge this with other pipes like dates or measurements